packages/playground/src/components/caprover_worker.vue

```diff
@@ -67,16 +69,17 @@
 </template>
 
 <script lang="ts" setup>
+import NodeSelector from "../components/node_selection.vue";
 import { useProfileManager } from "../stores";
 import rootFs from "../utils/root_fs";
-
 const emits = defineEmits<{ (event: "update:loading", value: boolean): void }>();
 const props = defineProps<{ modelValue: CaproverWorker }>();
 const rootFilesystemSize = computed(() =>
   rootFs(props.modelValue.solution?.cpu ?? 0, props.modelValue.solution?.memory ?? 0),
 );
 const profileManager = useProfileManager();
 const farmManager = useFarm();
+const selection = ref();
```

Comment:
Please add the type of this selection 

```suggestion
const selection = ref<SelectType>();
```